websqrl/src/MonacoEditor.tsx

```diff
@@ -32,6 +34,21 @@ export const MonacoEditor: React.FC<MonacoEditorProps> = ({
     editorRef.current?.updateOptions({ theme });
   }, [theme]);
 
+  useEffect(() => {
+    if (!editorRef.current || monacoEditorObj.state !== "success") return;
+    if (typeof navigator === "undefined") return;
+
+    const model = editorRef.current.getModel();
+    if (!model) return;
+
+    monacoEditorObj.value.editor.setModelMarkers(
+      model,
+      // i'm not sure what the owner should be
```

Comment:
This comment might be lost, should we resolve or remove it?
(or explain a little more)

Reply:
Looks like it’s an ID that monaco uses to namespace markers. If you set markers using that ID, it’ll overwrite existing ones with that ID. The `removeAllMarkers` also takes the ID as its only param.